griptape/engines/summary/prompt_summary_engine.py

```diff
@@ -53,7 +53,7 @@ def summarize_artifacts_rec(
             summary=summary, text=artifacts_text, rulesets=J2("rulesets/rulesets.j2").render(rulesets=rulesets)
         )
 
-        if self.prompt_driver.tokenizer.count_tokens_left(full_text) >= self.min_response_tokens:
+        if self.prompt_driver.tokenizer.count_input_tokens_left(full_text) >= self.min_response_tokens:
```

Comment:
Am I reasoning through this logic correct? This should be `count_input_tokens_left`, not `count_output_tokens_left`, right?

Reply:
I think you're right here, just based off the context of chunking the input and recursively summarizing.